app/helpers/application_helper.rb

```diff
@@ -7,4 +7,27 @@ def team_logo(image_path)
       radius: 5,
       border: '1px_solid_black')
   end
+
+  CLOUDINARY_JS_CONFIG_PARAMS = %i[
+    api_key,
+    cloud_name,
+    private_cdn,
+    secure_distribution,
```

Comment:
Found this bug while testing in staging, when using this syntax we should not include the comma:

```suggestion
    api_key
    cloud_name
    private_cdn
    secure_distributio
```